utils/isMaximumStorageLimitReached.ts

```diff
@@ -0,0 +1,18 @@
+import { Platform } from 'react-native';
+import {
+  getFreeDiskStorageOldSync,
+  getFreeDiskStorageSync,
+} from 'react-native-device-info';
+import { MAXIMUM_STORAGE_LIMIT_IN_MB } from 'react-native-dotenv';
+
+export default function isMaximumStorageLimitReached() {
+  const storageLimitInBytes = MAXIMUM_STORAGE_LIMIT_IN_MB * 1024 * 1024;
+
+  if (Platform.OS === 'android' && Platform.Version < 29) {
```

Comment:
can we refactor something like this

const freeDiskStorageInBytes = (Platform.OS === 'android' && Platform.Version < 29) ? getFreeDiskStorageOldSync() : getFreeDiskStorageSync();
  return freeDiskStorageInBytes <= storageLimitInBytes;

Reply:
Refactored